GETTING_STARTED.md

```diff
@@ -649,7 +649,7 @@ post.author.new_record? # => false
 To not save the associated object, specify `strategy: :build` in the factory:
 
 ```ruby
-FactoryBot.use_parent_strategy = false
+FactoryBot.use_parent_strategy = true
```

Comment:
Yeah, I think you are correct that something is off here. I think we could actually get rid of this line entirely, since `strategy: :build` takes precedence regardless of how `use_parent_strategy` is set.